lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -151,22 +155,46 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
       M.select_install_rm_cmd(cache_folder, project_name..'-tmp')
     }
   else
+    local git_folder
+    if is_windows then
+      git_folder = cache_folder ..'\\'.. project_name
+    else
+      git_folder = cache_folder ..'/'.. project_name
```

Comment:
there should be already a method giving you that folder.

Reply:
Please use `utils.join_path` here.